site/content/en/docs/api_sdk/cli/_index.md

```diff
@@ -7,20 +7,27 @@ description: ''
 
 ## Overview
 
-A simple command line interface for working with CVAT tasks. At the moment it
+A simple command line interface for working with CVAT. At the moment it
 implements a basic feature set but may serve as the starting point for a more
 comprehensive CVAT administration tool in the future.
 
 Overview of functionality:
 
-- Create a new task (supports name, bug tracker, project, labels JSON, local/share/remote files)
-- Delete tasks (supports deleting a list of task IDs)
-- List all tasks (supports basic CSV or JSON output)
-- Download JPEG frames (supports a list of frame IDs)
-- Dump annotations (supports all formats via format string)
-- Upload annotations for a task in the specified format (e.g. 'YOLO ZIP 1.0')
-- Export and download a whole task
-- Import a task
+- Projects:
```

Comment:
It would be great to have command names in this list, please consider adding them.

Reply:
Yeah, that's a good idea. Updated.